django_ipam/base/models.py

```diff
@@ -55,10 +55,12 @@ def request_ip(self, options=None):
 
 
 class AbstractIpAddress(TimeStampedEditableModel):
+    subnet = models.ForeignKey(swapper.get_model_name("django_ipam", "Subnet"),
+                               on_delete=models.CASCADE,
+                               help_text=_('Select a subnet and the first available IP address '
```

Comment:
can you find a way to add this help_text only in the admin and not in the model?
This help text only applies to the admin. If we leave it here it will be shown also in the browsable API interface when we will add the API and it will confuse our users.